vhdlparser/vhdlparser.jj

```diff
@@ -1012,14 +1012,15 @@ QCString entity_class() :  {}
 | <TYPE_T> {return "type";}
 | <SUBTYPE_T> {return "subtype";}
 | <CONSTANT_T> {return "constant";}
-| <SIGNAL_T> {return "signal";}
```

Comment:
Why is `SIGNAL_T` removed here as I see it is still used on other places?used

Reply:
This is the correct grammer from VHDL 2008

entity_class:
    | entity
    | architecture
    | configuration
    | procedure
    | function
    | package
    | type
    | subtype
    | constant
    | variable
    | component
    | label
    | literal
    | units
    | group
    | file
    | property
    | sequence